org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/DateRangePicker.java

```diff
@@ -112,17 +112,28 @@ public class DateRangePicker extends Popup implements EventListener<Event>, Valu
 	private Tabpanel toTabPanel;
 	private Div midDiv;
 	
+	/** date editor from **/
 	private WEditor editor;
+	/** date editor to **/
 	private WEditor editor2;
+	/** value of date from **/
 	private Date dateFrom;
+	/** value of date to **/
 	private Date dateTo;
-	private Date oldValueFrom;
-	private Date oldValueTo;
+	/** current/new display value **/
 	private String displayValue;
+	/** old display value **/
+	private String oldDisplayValue = "";
+	/** enable to fire value change listeners **/
 	private boolean enableValueChange = true;
```

Comment:
took me sometime to figure out that the javadoc comment for this variable is wrong. you are setting enableValueChange to false when you don't want to listen for ValueChangeEvent from editor and editor2.